### ISSUE TITLE
New navbar on mobile has overlap when using Spanish

### ISSUE THREAD

**Issue by iamkarlson** [TandoorRecipes/recipes#2382]
### Tandoor Version

1.4.8

### Setup

Docker / Docker-Compose

### Reverse Proxy

Nginx Proxy Manager (NPM)

### Bug description

In the new version, a bottom navbar has a small overlap with the last meal plan day. Nothing serious, just a papercut. 

![image](https://user-images.githubusercontent.com/4534374/227035479-05648371-cd72-469b-9ea9-153d8ee65af3.png)

**Comment by vabene1111** [comment]
do you think that there would be fitting but shorter translations for those menu entries so that they are shorter ?

**Comment by iamkarlson** [comment]
Indeed, that may fix the current problem, but you may pop up again for different languages, or [other problems](https://phrase.com/blog/posts/10-common-mistakes-in-software-localization/). I wouldn't go this way and just made the layout when they stack one upon another so it's never overlap.

**Comment by vabene1111** [comment]
yes i know, but thats easier said than done and no matter how much we move, it just looks ugly if the there is multiple rows of text.

As you now spanish better than I do, can you send me the proper translations or add them to translate.tandoor.dev so everyone can have a nicer looking nav?

**Comment by iamkarlson** [comment]
Thanks! Just registered. I will think about better translation but I'm not native and it won't yield any results anywhere soon.
